recipes/morphotactics/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "morphotactics" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dmort27/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 0c0ff9fc35acb9281438e4412f40c46f515cb1f0998a0ab301a0d1ce0c2a07bc
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win or py<36]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - cython >=0.29
+    - openfst ==1.8.1
+    - python
+    - pip
+    - setuptools >=42
+    - wheel
+  run:
+    - graphviz
+    - python
+    - pynini
+    - openfst ==1.8.1
```

* I don't think `graphviz` is used
* It looks like you only directly use `pynini` for openfst, so no need to directly list `openfst`?
* Should also list `pywrapfst` since it's importing that
* Need a python version for noarch
```suggestion
    - python >=3.6
    - pynini
    - pywrapfst
```